packages/eventwebhook/README.md

```diff
@@ -0,0 +1,38 @@
+# Install:
+```
+npm install --save @sendgrid/eventwebhook
+```
+
+# Handler webhook auth:
+```
+  //
+  // HANDLERS 
+  // 
+  var authHandler = (req, res, next) => {
+    const {EventWebhook}    = require('@sendgrid/eventwebhook')
+      , ew                  = new EventWebhook()
```

Comment:
Why are there commas here?

Reply:
My goal was not so much to explain how I program, but to contribute a piece of missing doc to the community after I spent lots of time figuring out how to authenticate Sendgrid's webhooks... 

This code-formatting allows:
. to get rid of redundant keywords like const, var, let;
. to make sure that a comma (by being at the start of each line) isn't forgotten between statements; and
. to appear as a bullet point that clarifies code reading.